### PR TITLE
Feature/second page/style

### PR DESCRIPTION
**Pull request by JessMotta**
I added styling on page two (show mentor info) and added some css files modulating sections in HTML.

### CODE REVIEW

**Comment by JessMotta** [comment]
I changed content's  width in screens larger than 1024px and font size.

File: scripts/calendar.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+function CalendarControl() {
+  const calendar = new Date();
+  const calendarControl = {
+    localDate: new Date(),
+    prevMonthLastDate: null,
+    calWeekDays: ["D", "S", "T", "Q", "Q", "S", "S"],
+    calMonthName: [
+      "Jan",
+      "FeV",
```
Comment (luanaggoncalves) [inline scripts/calendar.js:13]:
```suggestion
      "Fev",
```

**Review by luanaggoncalves — CHANGES_REQUESTED** [review]

File: scripts/calendar.js (hunk 1 above)
Comment (JessMotta) [reply scripts/calendar.js]: fixed

**Review by luanaggoncalves — APPROVED** [review]
Approved

**Review by jotape-SL — APPROVED** [review]

**Pull request merged**